route.js

```diff
@@ -17,15 +17,63 @@ app.listen(port, (err) => {
 
 
 app.get('/', (request, response) => {
-    response.send('Bienvenue sur Express');
+  response.send('Bienvenue sur Express');
 });
   
 app.get('/users', (req, res) => {
-    connection.query('SELECT * from DVM_Legal_Entity', (err, results) =>{
-        if(err) {
-            res.status(500).send('No body was here..')
-        } else {
-            res.json(results);
-        }
-    })
-});
\ No newline at end of file
+  connection.query('SELECT * from DVM_Legal_Entity', (err, results) =>{
+    if(err) {
+      res.status(500).send('No body was here..')
+    } else {
+      res.json(results);
+    }
+  })
+});
+
+app.get('/users/:id', (req, res) => {
+  const { id } = req.params
+  connection.query('SELECT * from DVM_Legal_Entity WHERE id = ?', id, (err, results) => {
+    if(err) {
+      res.status(500).send('Nobody was here..')
+    } else {
+      res.json(results);
+    }
+  })
+})
+
+app.post('/users', (req, res) => {
+  const formData = req.body;
```

Comment:
req.body may contain what you need to make an insertion, or it may not : you should handle 400 status code here